norns/play.md

```diff
@@ -143,21 +177,27 @@ Get out of MIDI-mapping mode by holding K1 and toggling off.
 
 On this page you can record and play directly to and from disk.
 
-K2 toggles focus between REC and PLAY.
+K2 toggles focus between PLAY and REC.
 
-REC
+PLAY
 
-- K3 to arm recording.
+- K3 loads file.
```

Comment:
nit: in preceding lists items are not period-delimited.  consider dropping the trailing `.`?